.changeset/config.json

```
@@ -0,0 +1,11 @@
+{
+  "$schema": "https://unpkg.com/@changesets/config@3.0.0/schema.json",
+  "changelog": "@changesets/cli/changelog",
+  "commit": false,
+  "fixed": [["@zazuko/yasgui-utils", "@zazuko/yasgui", "@zazuko/yasqe", "@zazuko/yasr"]],
```

FYI, globs are supported

```suggestion
  "fixed": [["@zazuko/*"]],
```